grammar.js

```diff
@@ -278,10 +278,15 @@ module.exports = grammar({
       ']',
     ),
 
-    polyvar_declaration: $ => prec.right(seq(
-      $.polyvar_identifier,
-      optional($.polyvar_parameters),
-    )),
+    polyvar_declaration: $ => prec.right(
+      choice(
+        seq(
+          $.polyvar_identifier,
+          optional($.polyvar_parameters),
+        ),
+        $._inline_type
```

Comment:
```suggestion
        $._type_identifier
```

Pretty sure a function type, object type, tuple type make no sense in this case